src/routes/auth.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const router = express.Router();
+const {User} = require('../models');
+const jwt = require('jsonwebtoken');
+const bcrypt = require("bcrypt");
+
+router.post('/login', (req, res) => {
+  const { email, password } = req.body;
+
+  User.findOne({ where: { email: email.toLowerCase() } })
+    .then(user => bcrypt.compare(password, user.password))
+    .then(match => {
+      if (!match) return res.status(401).json("Wrong Email or Password");
+      jwt.sign({user: req.user}, 'secretkey', (err, token) => {
+        res.status(200).json({
```

Comment:
don't forget to use ```return``` before res
If you will have another cases later it send response but function will try be finished.
So in this case
```if (whatever) res.status(200)```
```if (somethingElse) res.status(300)```
User will get 200 status and you will have server error. Cause it will try to set 300 status for response has already been sent.
But in this case function will be stopped as soon as it find ```return```
```if (whatever) return res.status(200)```
```if (somethingElse) return res.status(300)```